.github/workflows/maven-build.yml

```diff
@@ -116,6 +116,6 @@ jobs:
           distribution: 'zulu'
           cache: 'maven'
       - name: Deploy
-        run: mvn --batch-mode deploy
+        run: mvn deploy -Dmaven.test.skip -Dspotless.check.skip=true -Dgpg.skip -Dmaven.javadoc.skip=true -Djacoco.skip=true -Dspotbugs.skip  -Dmaven.surefire.debug
```

Comment:
are we cool with changing this? doesn't look like the CI publishing is working right now
https://www.notion.so/cortexio/How-to-publish-Github-Api-e80324a13abf4497afaa67e0a215e53a